src/functions-reference/array_operations.Rmd

```diff
@@ -409,3 +410,21 @@ Number of components of v less than v[s]
 `int` **`rank`**`(int[] v, int s)`<br>\newline
 Number of components of v less than v[s]
 
+## Reversing Functions {#reversing-functions}
+
+Reversing allows to create a copy of array in which elements are in reverse
+order. For example, if `v` is declared as a real array of size 3, with values
+\[ \text{v} = (1,\, -10.3,\, 20.987), \] then
+\[ \mathrm{reverse(v)} = (20.987,\, -10.3,\, 1). \]
+
+<!-- real[]; reverse; (real[] v); -->
+\index{{\tt \bfseries reverse }!{\tt (real[] v): real[]}|hyperpage}
+
+`real[]` **`reverse`**`(real[] v)`<br>\newline
+Create a copy of the array with the elements in reverse order.
+
+<!-- int[]; reverse; (int[] v); -->
+\index{{\tt \bfseries reverse }!{\tt (int[] v): int[]}|hyperpage}
+
+`int[]` **`reverse`**`(int[] v)`<br>\newline
+Create a copy of the array with the elements in reverse order.
```

Comment:
Does this only work for integer and real arrays?  It should work for any array if possible, or if not, at least all arrays up to a given number of dimensions.  If this PR doesn't create a generic `reverse` function, please create a new issue with that goal.  Then the doc can just use `T[] reverse (T[] x)`.

Reply:
The implementation does not restrict what can be reversed, so I'll change that to use `T[]`.